src/components/radio/README.md

```diff
@@ -54,6 +54,7 @@ The `md-radio-group` component has no button initially selected.
 | `value` | `any` | The value of this radio button. |
 | `checked` | `boolean` | Whether the radio is checked. |
 | `disabled` | `boolean` | Whether the radio is disabled. |
+| `actionName` | `string` | Used to set the `aria-label` attribute of the underlying input element. |
```

Comment:
We also using the `aria-label` input in the checkbox as well. (See [here](https://github.com/angular/material2/blob/master/src/components/checkbox/checkbox.ts#L80)).

Screenreaders shouldn't be confused about that attribute, because the host element is normally not focusable.

Without any attribute binding, you're still using the dash cased attribute
```html
<md-radio-button aria-label="My Label Input">
```

But with attribute binding, you're right, it will use the camelCased attribute name.